packages/restapi/src/lib/user/getGroup.ts

```diff
@@ -0,0 +1,23 @@
+import axios from 'axios';
```

Comment:
I think getGroup details should not be in user folder. Need some discussion on thi

Reply:
getGroup is needed in sdk also, so we can keep a seperate file for it